docs/user/openstack/troubleshooting.md

```diff
@@ -15,6 +15,20 @@ OpenStack CLI tools should be installed, then:
 
 Destroying the cluster has been noticed to [sometimes fail](https://github.com/openshift/installer/issues/1985). We are working on patching this, but in a mean time the workaround is to simply restart the destroying process of the cluster.
 
+## Machine has ERROR state
+
+This could be because the machine's instance was accidentally destroyed and the cluster API provider cannot recreate it.
+
+You can check the status of machines with the help of the command `oc get machines -n openshift-machine-api`.
```

Comment:
Keep code snippets consistent with the rest of the doc

````suggestion

You can check the status of machines with the command 

```sh
oc get machines -n openshift-machine-api
```
````